src/AlgebraicGeometry/Schemes/Sheaves/CoherentSheaves.jl

```diff
@@ -574,8 +574,8 @@ Coherent sheaf of modules
     1: [(y//x)]   affine 1-space
     2: [(x//y)]   affine 1-space
 with restrictions
-  1: direct sum of (Multivariate polynomial ring in 1 variable over GF(7)^1, Multivariate polynomial ring in 1 variable over GF(7)^1)
-  2: direct sum of (Multivariate polynomial ring in 1 variable over GF(7)^1, Multivariate polynomial ring in 1 variable over GF(7)^1)
+  1: direct sum of FreeMod{FqMPolyRingElem}[Multivariate polynomial ring in 1 variable over GF(7)^1, Multivariate polynomial ring in 1 variable over GF(7)^1]
+  2: direct sum of FreeMod{FqMPolyRingElem}[Multivariate polynomial ring in 1 variable over GF(7)^1, Multivariate polynomial ring in 1 variable over GF(7)^1]
```

Comment:
The changed docstring is caused by the attribute `direct_product` changing from a `Tuple` to a `Vector`, and it gets printed raw by this function:
```
function Hecke.show_direct_sum(io::IO, G)
  D = get_attribute(G, :direct_product)
  D === nothing && error("only for direct sums")
  print(io, "direct sum of ")
  show(IOContext(io, :compact => true), D)
end
```

For nicer printing, that function (and its siblings around it) should be improved.

That leave the question whether the change from tuple to vector is fine. I had a quick look at the code using this attribute and I believe it is all fine either. Long term a vector IMO makes more sense there.

CC @HechtiDerLachs